### PR TITLE
Support estimate gas consumption when executing the action from ioctl…

### PR DESCRIPTION
**Pull request by lzxm160**
… 1326
work on #1326

### CODE REVIEW

File: ioctl/cmd/action/action.go [hunk 1]
```diff
@@ -132,6 +156,15 @@ func execute(contract string, amount *big.Int, bytecode []byte) (err error) {
 		log.L().Error("error when invoke an execution", zap.Error(err))
 		return
 	}
+	if gasLimit == 300000 {
```
Comment (zjshen14) [inline ioctl/cmd/action/action.go]: why 300000?

**Comment by zjshen14** [comment]
else looks good

File: ioctl/cmd/action/action.go (hunk 1 above)
Comment (lzxm160) [reply ioctl/cmd/action/action.go]: Is there a better way to check if user sets gasLimitFlag?need to find out

Comment (zjshen14) [reply ioctl/cmd/action/action.go]: Default to 0, as long as 0, we will estimate?

Comment (lzxm160) [reply ioctl/cmd/action/action.go]: ok

**Pull request merged**